pythonProject/personally_adjusted_filter.py

```diff
@@ -0,0 +1,21 @@
+
+
+def my_filter(func, iterable):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/